api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -56,7 +56,17 @@ public Workspace apply(org.pmiops.workbench.db.model.Workspace workspace) {
               .commercialPurpose(workspace.getCommercialPurpose())
               .population(workspace.getPopulation())
               .populationOfFocus(workspace.getPopulationOfFocus())
-              .additionalNotes(workspace.getAdditionalNotes());
+              .additionalNotes(workspace.getAdditionalNotes())
+              .reviewRequested(workspace.getReviewRequested())
+              .approved(workspace.getApproved());
```

Comment:
Why does this need a null check in Swagger => db, but not here?

Reply:
I believe it doesn't actually need the null check in Swagger=>db here in particular (for approved). Will make the change now.